.swiftlint.yml

```diff
@@ -0,0 +1,35 @@
+disabled_rules: # rule identifiers turned on by default to exclude from running
+  - colon
+  - comma
+  - control_statement
+  - identifier_name
+  - cyclomatic_complexity
```

Comment:
Please also add these option rules.

```yml
opt_in_rules:
  - empty_count
  - let_var_whitespace
  - trailing_closure
  - let_var_whitespace
  - unused_declaration
  - unused_import
  - vertical_whitespace_closing_braces
```

Reply:
The empty_count rule is not so beautiful because it simply hits on the naming of count.
For example, it will error out for types that are not Arrays.
```
var count = 0
if count > 0 {} // ERROR: use `isEmpty`
```

I don't think `trailing_closure` should be a rule, since it reduces readability in some cases.

Added other rules. https://github.com/ca-dp/bucketeer-ios-sdk/pull/96/commits/3746d965f0d9b994729d517d8b086b8dddf13d12